src/aead/aead.rs

```diff
@@ -24,6 +24,71 @@
 //!
 //! [AEAD]: http://www-cse.ucsd.edu/~mihir/papers/oem.html
 //! [`crypto.cipher.AEAD`]: https://golang.org/pkg/crypto/cipher/#AEAD
+//!
+//! # Examples
+//!
+//!```rust
+//!extern crate ring;
+//!
+//!use ring::aead::*;
+//!use ring::rand::SystemRandom;
```

Comment:
And actually, cargo test works on my machine, I will try to figure out before changing it.